hudi-common/src/main/java/org/apache/hudi/common/fs/TimedFSInputStream.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
```

Comment:
Why still name the class `TimedFSInputStream` based on the fact that the written bytes size is also recorded ?

Reply:
This class does not record the written bytes.